Makefile

```diff
@@ -11,15 +11,28 @@ create_project:
 # Y - in response to Do you want to generate the .github/ISSUE_TEMPLATE and .github/PULL_REQUEST_TEMPLATE?
 # Y - in response to Do you want to generate the Github Action workflow?
 # Y - in response to Would you like to add the Oban addon?
+common_addon_prompts = Y\nY\nY\nY\n
+
+# Y - in response to Would you like to add the CoreJS addon?
```

Comment:
Is there really a case when CoreJS would not be needed? Shouldn't bt installed by default for a Web project since all projects would use JS?

Reply:
Actually, I have had the same thoughts too, haha, will set it to default addon for Web variant 👍

Reply:
Fixed in https://github.com/nimblehq/elixir-templates/pull/58/commits/4785e4f3f64137f8ba4cfc0d5f8cb8bbc25ece81